test/UnitTest/TestingControllers/InstancesControllerTests.cs

```diff
@@ -622,6 +623,87 @@ public async Task GetEmptyListOfInstances_UserRequestsInstancesNoPartyIdDefinedA
+        /// </summary>
+        [Fact]
+        public async Task GetMany_UserRequestsInstancesNoPartyIdDefinedAndWithWrongOrganisation_ThrowsBadRequest()
+        {
+            // Arrange
+            string requestUri = $"{BasePath}";
+            int partyId = 1337;
+            string expectedResponseMessage = "Organisation number needs to be exactly 8 or 9 digits.";
+
+            Mock<IRegisterService> registerService = new Mock<IRegisterService>();
+            registerService.Setup(x => x.PartyLookup(It.Is<string>(p => p == null), It.Is<string>(o => o == "333123213"))).Throws(new BadHttpRequestException(expectedResponseMessage));
+
+            HttpClient client = GetTestClient(null, registerService);
+            string token = PrincipalUtil.GetToken(3, partyId);
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+            client.DefaultRequestHeaders.Add("X-Ai-InstanceOwnerIdentifier", "Organisation:333123213");
+
+            // Act
+            HttpResponseMessage response = await client.GetAsync(requestUri);
+            string responseMessage = await response.Content.ReadAsStringAsync();
+
+            // Assert
+            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+            registerService.VerifyAll();
+        }
+
+        /// <summary>
+        /// Test case: Get Multiple instances with invalid organisation number and without specifying instance owner partyId.
+        /// Expected: Controller throws 400 bad request.
+        /// </summary>
+        [Fact]
+        public async Task GetMany_UserRequestsInstancesNoPartyIdDefinedAndWithInvalidOrganisation_ThrowsBadRequest()
+        {
+            // Arrange
+            string requestUri = $"{BasePath}";
+            int partyId = 1337;
+            string expectedResponseMessage = "Organisation number needs to be exactly 8 or 9 digits.";
+
+            HttpClient client = GetTestClient(null, null);
+            string token = PrincipalUtil.GetToken(3, partyId);
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+            client.DefaultRequestHeaders.Add("X-Ai-InstanceOwnerIdentifier", "Organisation:33312");
+
+            // Act
+            HttpResponseMessage response = await client.GetAsync(requestUri);
+            string responseMessage = await response.Content.ReadAsStringAsync();
+
+            // Assert
+            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+            Assert.Contains(expectedResponseMessage, responseMessage);
+        }
+
+        /// <summary>
+        /// Test case: Get Multiple instances with invalid organisation number and without specifying instance owner partyId.
+        /// Expected: Controller throws 400 bad request.
+        /// </summary>
+        [Fact]
+        public async Task GetMany_UserRequestsInstancesNoPartyIdDefinedAndWithInvalidPerson_ThrowsBadRequest()
```

Comment:
```suggestion
        public async Task GetMany_UserRequestsInstancesNoPartyIdDefinedAndWithInvalidPerson_ReturnsBadRequest()
```

No exception involved, use returns instead of throws